basedtyping/runtime_only.py

```diff
@@ -4,10 +4,13 @@
 This is the similar to the ``types`` module.
 """
 
-from typing import Final, Literal, Union
+from typing import Type, Union
 
-LiteralType: Final = type(Literal[1])
+from typing_extensions import Final, Literal
+
+LiteralType: Final[type] = type(Literal[1])  # type: ignore[no-any-expr]
 """A type that can be used to check if type hints are a ``typing.Literal`` instance"""
 
-OldUnionType: Final = type(Union[str, int])
+# TODO: this is type[object], we need it to be 'SpecialForm[Union]' (or something)
+OldUnionType: Final[Type[object]] = type(Union[str, int])
```

Comment:
is this another thing you were testing you didnt mean to commit?
```suggestion
OldUnionType: Final = type(Union[str, int])
```

Reply:
Actually that annotation is removing a `type[Any]`.